CHANGES.md

```diff
@@ -1,6 +1,43 @@
 
 # Changes
 
+## edge-21.1.3
+
+This edge release improves proxy diagnostics and handling for the failfast state
+by improving log output and enhancing the proxy's ability to recover when it is
+in failfast. In addition, the commands and sub-commands for `linkerd viz` have
+been updated to be consistent with the `linkerd` core commands. Also, the
+install commands for the core and multicluster components have been modified to
+accept helm-style customization flags.
+
+Full release notes:
+
+* Added helm `set`, `set-string`, `values`, `set-files` customization flags for
+  the `linkerd install` and `linkerd multicluster install` commands
+* Fixed `linkerd metrics` resource selector to differentiate between owner
```

Comment:
Fixed an issue where `linkerd metrics` could return metrics for the incorrect set of pods when there are overlapping label selectors